### PR TITLE
NPC natural weapon/equipment differentiation

### PR DESCRIPTION
**Pull request by ian612**
This update adds a tag to weapon items to indicate whether they are equipment or not, and filters the NPC inventory tab to remove items not tagged as equipment. This removes natural weapons and unarmed strikes from thhe inventory tab, while still leaving them accessible for editing and use. By default in template.json, the _IsEquipment_ value is true, and should introduce no breaking changes from a data entry standpoint.

The checkbox for determining this property is added only to item sheets whose type is "weapon".

image
image

For weapons that are marked as "Weapon is Equipment" (default), they appear in the NPC "Features" and "Inventory" tabs, as normal. For weapons with this box unchecked, they only appear in the "Features" tab.

image

On PC sheets, no changes are made. All weapons appear on the inventory tab, still.

image

This tag should also allow better interaction with modules like ItemPiles that allow players to "loot" equipment from fallen enemies.

### CODE REVIEW

**Comment by LebombJames** [comment]
What's the change in the picture of the armor sheet?

File: src/less/items.css [hunk 1]
```diff
@@ -0,0 +1,660 @@
+:root {
```
Comment (LebombJames) [inline src/less/items.css:1]: I don't think this is meant to be here

**Comment by LebombJames** [comment]
Could you also take some screenshots of the new tabs on equipment? Not really able to crack open your branch and test myself rn.

File: src/less/items.css (hunk 1 above)
Comment (ian612) [reply src/less/items.css]: Yeah you're correct, dunno how this ended up there. I've removed it

**Comment by ian612** [comment]
> 



> What's the change in the picture of the armor sheet?

There's no change, just included it to show that the new checkbox only appears on weapon sheets.

**Comment by ian612** [comment]
> 



> Could you also take some screenshots of the new tabs on equipment? Not really able to crack open your branch and test myself rn.

I haven't added the new tabs yet. This was something else I was working on at the same time that was basically done, so I broke it out into a separate pull request to keep everything segmented. I'll open a draft PR for the new tabs thing.

**Comment by LebombJames** [comment]
> > 
> 
> > Could you also take some screenshots of the new tabs on equipment? Not really able to crack open your branch and test myself rn.
> 
> I haven't added the new tabs yet. This was something else I was working on at the same time that was basically done, so I broke it out into a separate pull request to keep everything segmented. I'll open a draft PR for the new tabs thing.

The tabs look to be in this PR, see the changes to `weapon.hbs`

**Comment by ian612** [comment]
> > > 
> > 
> > 
> > > Could you also take some screenshots of the new tabs on equipment? Not really able to crack open your branch and test myself rn.
> > 
> > 
> > I haven't added the new tabs yet. This was something else I was working on at the same time that was basically done, so I broke it out into a separate pull request to keep everything segmented. I'll open a draft PR for the new tabs thing.
> 
> The tabs look to be in this PR, see the changes to `weapon.hbs`

Yep you're correct. I branched this off the subtabs branch instead of off development like I should have, so it's a bit messy. I'll try to clean this up to keep the subtabs separated, as they're not necessary for this.

File: src/templates/items/parts/item-status.hbs [hunk 2]
```diff
@@ -6,6 +6,14 @@
                     <input type="checkbox" name="system.proficient" {{checked itemData.proficient}}/> {{ localize "SFRPG.Items.Proficient" }}
                 </label>
                 {{/if}}
+                {{#if isWeapon}}
+                <label class="checkbox">
+                    <input type="checkbox" name="system.isEquipment" {{checked itemData.isEquipment}}/> {{ localize "SFRPG.Items.Weapon.IsEquipment" }}
```
Comment (LebombJames) [inline src/templates/items/parts/item-status.hbs:11]: It'd be nice to have a tooltip on this describing the use case and behaviour of this feature

File: src/templates/items/weapon.hbs [hunk 3]
```diff
@@ -61,7 +61,7 @@
 
             {{!-- Special Materials Template --}}
             {{> "systems/sfrpg/templates/items/parts/item-special-materials.hbs"}}
-
+
```
Comment (LebombJames) [inline src/templates/items/weapon.hbs:64]: Errant whitespace

**Review by LebombJames — CHANGES_REQUESTED** [review]

File: src/templates/items/parts/item-status.hbs (hunk 2 above)
Comment (ian612) [reply src/templates/items/parts/item-status.hbs]: Agreed, I've added one with a description of what checking it signifies

**Comment by LebombJames** [comment]
Could you also update the other localisations while you're at it?

**Comment by ian612** [comment]
> Could you also update the other localisations while you're at it?

You mean add tooltips to the other equipment statuses?

**Comment by LebombJames** [comment]
> > Could you also update the other localisations while you're at it?
> 
> You mean add tooltips to the other equipment statuses?

No just copy the one you added to English to the others

**Comment by ian612** [comment]
> > > Could you also update the other localisations while you're at it?
> > 
> > 
> > You mean add tooltips to the other equipment statuses?
> 
> No just copy the one you added to English to the others

Ahhh gotcha. Done.

**Pull request merged**